<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="VPA Login Tests"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="46.671">
    <properties></properties>

            <testcase
                name="itHoacNhieuhon10So"
                time="17.916"
                classname="VPA_Android.Login.ValidateForgotSDT">

            <failure type="java.lang.AssertionError" message="Timeout chờ element hiển thị: By.xpath: //android.widget.TextView[@resource-id=&quot;com.vpa.daugia:id/baseEditTextError&quot;]
Screenshot: C:\Users\Admin\IdeaProjects\alo\test-output\screenshots\VPA\FAIL_element_timeout_20241107_222329.png">
                <![CDATA[java.lang.AssertionError: Timeout chờ element hiển thị: By.xpath: //android.widget.TextView[@resource-id="com.vpa.daugia:id/baseEditTextError"]
Screenshot: C:\Users\Admin\IdeaProjects\alo\test-output\screenshots\VPA\FAIL_element_timeout_20241107_222329.png
                org.testng.Assert.fail(Assert.java:110)base.ValidateHelpers.assertElementText(ValidateHelpers.java:918)VPA_Android.Login.ValidateForgotSDT.itHoacNhieuhon10So(ValidateForgotSDT.java:41)java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)java.base/java.lang.reflect.Method.invoke(Method.java:580)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)java.base/java.util.ArrayList.forEach(ArrayList.java:1597)org.testng.TestRunner.privateRun(TestRunner.java:829)org.testng.TestRunner.run(TestRunner.java:602)org.testng.SuiteRunner.runTest(SuiteRunner.java:437)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)org.testng.SuiteRunner.run(SuiteRunner.java:330)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)org.testng.TestNG.runSuitesLocally(TestNG.java:1176)org.testng.TestNG.runSuites(TestNG.java:1099)org.testng.TestNG.run(TestNG.java:1067)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
            </failure>
    </testcase>


            <testcase
                name="nhapKhongPhaiLaSo"
                time="21.333"
                classname="VPA_Android.Login.ValidateForgotSDT">
            </testcase>
            <testcase
                name="sdtHopLe"
                time="7.422"
                classname="VPA_Android.Login.ValidateForgotSDT">
            </testcase>
</testsuite>
